Description: .DS_Store files are stripped out by dh_clean.
Author: Vincent Bernat <bernat@debian.org>
Forwarded: no

Index: cookiecutter-0.6.4/tests/test_generate.py
===================================================================
--- cookiecutter-0.6.4.orig/tests/test_generate.py	2013-08-21 12:33:41.000000000 +0200
+++ cookiecutter-0.6.4/tests/test_generate.py	2013-09-29 13:05:13.552869050 +0200
@@ -67,7 +67,6 @@
             template_dir='tests/input{{binary_test}}'
         )
         self.assertTrue(os.path.isfile('tests/inputbinary_files/logo.png'))
-        self.assertTrue(os.path.isfile('tests/inputbinary_files/.DS_Store'))
         self.assertTrue(os.path.isfile('tests/inputbinary_files/readme.txt'))
         self.assertTrue(
             os.path.isfile('tests/inputbinary_files/some_font.otf')
@@ -79,7 +78,6 @@
             template_dir='tests/input{{binary_test}}'
         )
         expected = ['tests/inputbinary_files/binary_files/logo.png',
-                    "tests/inputbinary_files/binary_files/.DS_Store",
                     "tests/inputbinary_files/binary_files/readme.txt"]
         for each in expected:
             self.assertTrue(os.path.isfile(each))
